### PR TITLE
PT test: Add 20 PT cases with perf tool.

### PR DESCRIPTION
**Pull request by ammyyi**
Signed-off-by: Ammy Yi <ammy.yi@intel.com>

### CODE REVIEW

**Comment by ammyyi** [comment]
These 2 failures cannot be fixed. Since if we add "", it will cause perf script error.

pt/perf_tests.sh:370 - Double quote to prevent globbing and word splitting., refer to https://github.com/koalaman/shellcheck/wiki/SC2086
pt/perf_tests.sh:371 - Double quote to prevent globbing and word splitting., refer to https://github.com/koalaman/shellcheck/wiki/SC20[8](https://github.com/intel/lkvs/actions/runs/3756710385/jobs/6383085132#step:4:9)6

**Comment by ysun** [comment]
We can force merge it, in case the check error is hard to be fix.
Please review and comment @hanning0511

**Comment by hanning0511** [comment]
> These 2 failures cannot be fixed. Since if we add "", it will cause perf script error.
> 
> pt/perf_tests.sh:370 - Double quote to prevent globbing and word splitting., refer to https://github.com/koalaman/shellcheck/wiki/SC2086 pt/perf_tests.sh:371 - Double quote to prevent globbing and word splitting., refer to [https://github.com/koalaman/shellcheck/wiki/SC20[8](https://github.com/intel/lkvs/actions/runs/3756710385/jobs/6383085132#step:4:9)6](https://github.com/koalaman/shellcheck/wiki/SC20%5B8%5D(https://github.com/intel/lkvs/actions/runs/3756710385/jobs/6383085132#step:4:9)6)

Agree

File: pt/README.md [hunk 1]
```diff
@@ -46,3 +46,7 @@ Non root user do snapshot trace check.
 
 ## Expected result
 All test results should show pass, no fail.
+
+Usage for perf_tests.sh:
+1. Make sure latest perf is there.
+2. Run case as ./perf_tests.sh -t casename
```
Comment (hanning0511) [inline pt/README.md:52]: Little tip: using inline code block.

Run case as `./perf_tests.sh -t <case name>`.

Comment (ammyyi) [reply pt/README.md]: Updated.

**Review by hanning0511 — APPROVED** [review]

**Pull request merged**